tools/depends/target/Toolchain.cmake.in

```diff
@@ -10,7 +10,10 @@ if(OS STREQUAL linux)
   set(CMAKE_SYSTEM_NAME Linux)
   set(CORE_SYSTEM_NAME linux)
   if(PLATFORM STREQUAL raspberry-pi)
-    set(CORE_SYSTEM_NAME rbpi)
+    set(CORE_PLATFORM_NAME_LC rbpi)
```

Comment:
bad rebase? 
please compare this with Toolchain.cmake.in from my commit: https://github.com/wsnipex/xbmc/commit/959f69bf3071d50bfa4a3fb74271bab2edf4890e

same below.